webpack.config.js

```diff
@@ -192,6 +197,10 @@ const webpackConfig = ( mode ) => {
 					name: 'Plugin CSS',
 					color: '#4285f4',
 				} ),
+				new CleanWebpackPlugin( {
+					// Stop this build from removing unrelated JS and CSS files.
```

Comment:
I'm assuming this means "unrelated to webpack", but I think this comment could be clearer as to which files we're ignoring and why 😄

Reply:
@tofumatt @aaemnnosttv Sorry for the unclear language here!  Without this option added, each build removes items created by the build before it because they are unrelated to the build being run.